src/CAD/Prism.hh

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "opengl/Shape.hh"
+#incldue "opengl/Polygon.hh"

```

Comment:
I don't think this is being used in the project given the misspelled include, so I might end up removing this file.